### PR TITLE
Add a liveness probe to the frr container

### PR DESCRIPTION
**Pull request by fedepaol**
In order to make sure that frr is up and running, we add a liveness probe to check that the daemons are available.

### CODE REVIEW

File: config/manifests/metallb-frr.yaml [hunk 1]
```diff
@@ -1927,6 +1927,12 @@ spec:
         - name: TINI_SUBREAPER
           value: "true"
         image: frrouting/frr:v7.5.1
+        livenessProbe:
+          exec:
+            command:
+            - /etc/frr_liveness/liveness.sh
+          initialDelaySeconds: 5
```
Comment (oribon) [inline config/manifests/metallb-frr.yaml:1934]: this seems a bit short ( as a guess ), is there a reason you went with 5 seconds?
also a startupprobe might be more fitting, wdyt?

Comment (fedepaol) [reply config/manifests/metallb-frr.yaml:1934]: yeah, it makes sense. Will change

Comment (fedepaol) [reply config/manifests/metallb-frr.yaml]: done

File: config/frr/speaker-patch.yaml [hunk 2]
```diff
@@ -78,6 +89,16 @@ spec:
...
               tail -f /etc/frr/frr.log
+          livenessProbe:
+            exec:
+              command: ["/etc/frr_liveness/liveness.sh"]
+            periodSeconds: 5
+            failureThreshold: 3
+          startupProbe:
+            exec:
+              command: ["/etc/frr_liveness/liveness.sh"]
+            failureThreshold: 30
```
Comment (liornoy) [inline config/frr/speaker-patch.yaml:100]: is it 30 on purpose here? or should it be 3?

Comment (fedepaol) [reply config/frr/speaker-patch.yaml:100]: it's on purpouse

File: charts/metallb/templates/speaker.yaml [hunk 3]
```diff
@@ -341,6 +352,16 @@ spec:
         resources:
           {{- toYaml . | nindent 12 }}
         {{- end }}
+        livenessProbe:
+          exec:
+            command: ["/etc/frr_liveness/liveness.sh"]
+          initialDelaySeconds: 5
+          periodSeconds: 5
```
Comment (liornoy) [inline charts/metallb/templates/speaker.yaml:359]: Shouldn't this be: 
            periodSeconds: 5
            failureThreshold: 3
like in different places in the code?

Comment (fedepaol) [reply charts/metallb/templates/speaker.yaml:359]: yeah that's the old version, good catch

Comment (fedepaol) [reply charts/metallb/templates/speaker.yaml]: fixed

**Comment by liornoy** [comment]
LGTM, also tested on my Kind cluster and works great.

**Pull request merged**